### PR TITLE
DB-2625: [next-drupal-starter] Document common issues

### PR DESCRIPTION
**Pull request by CobyPear**
## What changes were made?
- Add a Troubleshooting Local Development doc

## Where were the changes made?
docs

## Additional information

Don't forget to [add a changeset](https://github.com/pantheon-systems/decoupled-kit-js#generating-a-changeset) if needed!

### CODE REVIEW

**Review by backlineint — CHANGES_REQUESTED** [review]
A little hard to predict, but I could see cases where troubleshooting might be more general, rather than related to local development. A specific type of query or content integration that people often get wrong for example. The last portion of the images entry kind of falls into this category as well.

What do you think about maybe just renaming this section 'Troubleshooting'?

**Comment by CobyPear** [comment]
> What do you think about maybe just renaming this section 'Troubleshooting'?

I think that is reasonable. I was being a bit too careful about keeping this doc away from any platform considerations, but I see what you mean.

File: web/docs/Frontend Starters/Next.js/Next.js + Drupal/troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+id: 'next-drupal-troubleshooting'
+title: 'Troubleshooting' 
+sidebar_position: 5
+slug: '/Frontend Starters/Next.js/Next.js + Drupal/Troubleshooting Local Development'
```
Comment (backlineint) [inline web/docs/Frontend Starters/Next.js/Next.js + Drupal/troubleshooting.md]: The slug should probably change here now that this is just 'Troubleshooting'

**Review by backlineint — APPROVED** [review]
I pushed up a minor tweak for the slug here.

**Pull request merged**